pyproject.toml

```
@@ -108,5 +103,10 @@ module = [
 ]
 ignore_errors = true
 
+[tool.poetry.plugins."console_scripts"]
+"version-check" = "exasol.toolbox.pre_commit_hooks.package_version:main"
+
+
 [tool.poetry.scripts]
 tbx = 'exasol.toolbox.tools.tbx:CLI'
+sphinx-multiversion = 'exasol.toolbox.sphinx.multiversion:main'
```

Could the name have possible conflicts?
Maybe better:
```suggestion
exa-sphinx-multiversion = 'exasol.toolbox.sphinx.multiversion:main'
```